### PR TITLE
Itp 647

### PR DESCRIPTION
**Pull request by dalin-aranga**
sign part without sagas and redux

### CODE REVIEW

File: ramp-up-web/src/app/saga.js [hunk 1]
```diff
@@ -1,31 +1,57 @@
...
+  updateStudent,
   deleteStudent,
   getStudents,
-  updateStudent,
 } from "../utils/services";
+import { findUser } from "../utils/userService";
 
-// eslint-disable-next-line no-unused-vars
-function* WatchStudentAdded({ payload: post }) {
+function* WatchGetStudent() {
```
Comment (Dulajdeshan) [inline ramp-up-web/src/app/saga.js:13]: Function name should start with lowercase

File: ramp-up-web/src/app/saga.js [hunk 2]
```diff
@@ -1,31 +1,57 @@
...
 }
+function* watchAddStudent({ payload: payload }) {
+  try {
+    yield call(insertStudent, payload);
+  } catch (error) {
+    console.log(error);
+  }
+}
+function* watchUpdateStudent({ payload: payload }) {
+  console.log("Update Data ", payload);
```
Comment (Dulajdeshan) [inline ramp-up-web/src/app/saga.js:29]: Remove logs

File: ramp-up-web/src/components/myCommandCell.js [hunk 3]
```diff
@@ -2,7 +2,8 @@ import * as React from "react";
 export const MyCommandCell = (props) => {
```
Comment (Dulajdeshan) [inline ramp-up-web/src/components/myCommandCell.js:2]: File name should start with uppercase.

File: ramp-up-web/src/pages/Grid.js [hunk 4]
```diff
@@ -9,27 +10,37 @@ import {
...
+  const dispatch = useDispatch();
+  // eslint-disable-next-line no-unused-vars
   const [data, setData] = useState([]);
+
   useEffect(() => {
-    getStudents().then(({ data }) => setData(data));
+    dispatch(studentSlice.actions.getStudents());
   }, []);
 
+  const students = useSelector((state) => state.students.students);
```
Comment (Dulajdeshan) [inline ramp-up-web/src/pages/Grid.js:34]: Move this before the `useEffect`

File: ramp-up-web/src/pages/Grid.js [hunk 5]
```diff
@@ -48,34 +59,43 @@ const GridUI = () => {
...
-      console.log("data birthday", res.data.date);
-      const newData = { ...res.data };
-
-      const oldStudents = data;
-      oldStudents.pop(newData);
-      setData([newData, ...oldStudents]);
-    });
+    dispatch(studentSlice.actions.createStudent({ dataItem }));
+    window.location.reload(false);
+    // console.log("ID data", dataItem);
```
Comment (Dulajdeshan) [inline ramp-up-web/src/pages/Grid.js:77]: Remove unwanted comments

File: ramp-up-web/src/pages/Grid.js [hunk 6]
```diff
@@ -48,34 +59,43 @@ const GridUI = () => {
...
-    deleteStudent(dataItem).then(() => {
-      getStudents().then((data) => {
-        setData(data);
-      });
-      socket.emit("student_remove", `Student ${dataItem.id} was delete`);
-      getStudents();
-    });
+    dispatch(studentSlice.actions.deleteStudent(dataItem));
+    window.location.reload(false);
+    // deleteStudent(dataItem).then(() => {
```
Comment (Dulajdeshan) [inline ramp-up-web/src/pages/Grid.js:64]: remove unwanted comments

File: ramp-up-web/src/pages/Grid.js [hunk 7]
```diff
@@ -89,15 +109,18 @@ const GridUI = () => {
...
   const enterEdit = (dataItem) => {
-    console.log("Enter Edit", dataItem);
+    console.log("Data Edit", data);
+    console.log("Data Edit data item", dataItem);
     const newData = data.map((item) =>
       item.id === dataItem.id
         ? { ...item, date: new Date(item.date), inEdit: true }
         : item
     );
+    console.log("Edit Button", newData);
```
Comment (Dulajdeshan) [inline ramp-up-web/src/pages/Grid.js:119]: Remove logs

File: ramp-up-web/src/pages/Grid.js [hunk 8]
```diff
@@ -89,15 +109,18 @@ const GridUI = () => {
   };
 
   const enterEdit = (dataItem) => {
-    console.log("Enter Edit", dataItem);
+    console.log("Data Edit", data);
```
Comment (Dulajdeshan) [inline ramp-up-web/src/pages/Grid.js:112]: Remove logs

File: ramp-up-web/src/pages/Signin.js [hunk 9]
```diff
@@ -15,7 +15,9 @@ import { makeStyles } from "@material-ui/core/styles";
 import { findUser } from "../utils/userService";
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
-
+// import { useEffect } from "react";
```
Comment (Dulajdeshan) [inline ramp-up-web/src/pages/Signin.js:18]: Remove commented imports

**Comment by dalin-aranga** [comment]
"File name should start with uppercase." My vs code file name show as MyCommandCell but in github it shows myCommandCell like way

**Comment by dalin-aranga** [comment]
Done

**Pull request closed without merging**